accepted/PSR-4-autoloader-examples.md

```diff
@@ -141,7 +145,7 @@ class Psr4AutoloaderClass
         $prefix = trim($prefix, '\\') . '\\';
 
         // normalize the base directory with a trailing separator
-        $base_dir = rtrim($base_dir, DIRECTORY_SEPARATOR) . '/';
+        $base_dir = rtrim($base_dir, '\/') . '/';
```

Comment:
Why changing `DIRECTORY_SEPARATOR` into this?

Reply:
@drupol need use it everywhere or nowhere

Reply:
But then using `DIRECTORY_SEPARATOR` is the way to go, no ?

Reply:
Thanks, I've learned something today :)

Reply:
@drupol for the same reason, I keeped useless check `false !==` in line `while (false !== $pos = strrpos($prefix, '\\')) {`

Reply:
@drupol sorry, my fail, i'm gave invalid answer -  it's using to normalize Windows paths too